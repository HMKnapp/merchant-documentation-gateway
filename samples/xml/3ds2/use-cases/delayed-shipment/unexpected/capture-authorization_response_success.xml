<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<payment xmlns="http://www.elastic-payments.com/schema/payment" self="https://api-test.wirecard.com:443/engine/rest/merchants/33f6d473-3036-4ca5-acb5-8c64dac862d1/payments/f1f2c6f8-798e-45e6-a85d-a5a28f97ce4d">
    <merchant-account-id ref="https://api-test.wirecard.com:443/engine/rest/config/merchants/33f6d473-3036-4ca5-acb5-8c64dac862d1">33f6d473-3036-4ca5-acb5-8c64dac862d1</merchant-account-id>
    <transaction-id>f1f2c6f8-798e-45e6-a85d-a5a28f97ce4d</transaction-id>
    <request-id>fc65f023-5923-40ce-9868-c6fcfe72f5ea</request-id>
    <transaction-type>capture-authorization</transaction-type>
    <transaction-state>success</transaction-state>
    <completion-time-stamp>2020-02-11T12:52:35.000Z</completion-time-stamp>
    <statuses>
        <status code="201.0000" description="3d-acquirer:The resource was successfully created." severity="information"/>
    </statuses>
    <requested-amount currency="EUR">10.00</requested-amount>
    <parent-transaction-id>0fc5650c-c6f4-49b7-8362-8da088346562</parent-transaction-id>
    <account-holder>
        <first-name>Paul</first-name>
        <last-name>Peterson</last-name>
        <email>ppeter@example.com</email>
        <address>
            <street1>978C Golf Street</street1>
            <city>Round Lake</city>
            <state>IL</state>
            <country>US</country>
            <postal-code>60073</postal-code>
        </address>
        <merchant-crm-id>XWLfrzuGhpajYxFiFgNScwcLhtIOvXNxMIcrxIPTHqgroaFiPNDqMxGBXYUhMVBZ</merchant-crm-id>
    </account-holder>
    <shipping>
        <first-name>Paul</first-name>
        <last-name>Peterson</last-name>
        <address>
            <street1>978C Golf Street</street1>
            <city>Round Lake</city>
            <state>IL</state>
            <country>US</country>
            <postal-code>60073</postal-code>
        </address>
        <shipping-method>home_delivery</shipping-method>
    </shipping>
    <card>
        <expiration-month>1</expiration-month>
        <expiration-year>2023</expiration-year>
        <card-type>mastercard</card-type>
    </card>
    <card-token>
        <token-id>5489620654931192</token-id>
        <masked-account-number>541333******1192</masked-account-number>
    </card-token>
    <ip-address>127.0.0.1</ip-address>
    <order-detail>66a3281e-ef43-537c-d9e9-aee73c41a9fc</order-detail>
    <descriptor>3DS-2 Use-Cases: Delayed Shipment - Un-Expected Dalay</descriptor>
    <notifications format="application/json">
        <notification url="http://www.webhook.site/68f1326e-311d-4ef8-a998-1c3c66cc497a"></notification>
    </notifications>
    <payment-methods>
        <payment-method name="creditcard"/>
    </payment-methods>
    <parent-transaction-amount currency="EUR">10.000000</parent-transaction-amount>
    <authorization-code>424088</authorization-code>
    <api-id>elastic-api</api-id>
    <provider-account-id>70003</provider-account-id>
</payment>