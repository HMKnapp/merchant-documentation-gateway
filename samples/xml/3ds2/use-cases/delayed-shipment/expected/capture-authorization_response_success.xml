<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<payment xmlns="http://www.elastic-payments.com/schema/payment" self="https://api-test.wirecard.com:443/engine/rest/merchants/33f6d473-3036-4ca5-acb5-8c64dac862d1/payments/c927e8d4-6a6d-45aa-94dc-4ea85fc410b1">
    <merchant-account-id ref="https://api-test.wirecard.com:443/engine/rest/config/merchants/33f6d473-3036-4ca5-acb5-8c64dac862d1">33f6d473-3036-4ca5-acb5-8c64dac862d1</merchant-account-id>
    <transaction-id>c927e8d4-6a6d-45aa-94dc-4ea85fc410b1</transaction-id>
    <request-id>2984bb52-43c9-485d-bad6-6ef6e9ecf780</request-id>
    <transaction-type>capture-authorization</transaction-type>
    <transaction-state>success</transaction-state>
    <completion-time-stamp>2020-02-12T10:29:34.000Z</completion-time-stamp>
    <statuses>
        <status code="201.0000" description="3d-acquirer:The resource was successfully created." severity="information"/>
        <status code="200.1083" description="3d-acquirer:Cardholder Successfully authenticated." severity="information"/>
    </statuses>
    <requested-amount currency="EUR">10.00</requested-amount>
    <parent-transaction-id>739785bd-4b43-4d13-853c-3bef468ae8c0</parent-transaction-id>
    <account-holder>
        <first-name>Paul</first-name>
        <last-name>Peterson</last-name>
        <email>ppeter@example.com</email>
        <address>
            <street1>978C Golf Street</street1>
            <city>Round Lake</city>
            <state>IL</state>
            <country>US</country>
            <postal-code>60073</postal-code>
        </address>
        <merchant-crm-id>XWLfrzuGhpajYxFiFgNScwcLhtIOvXNxMIcrxIPTHqgroaFiPNDqMxGBXYUhMVBZ</merchant-crm-id>
    </account-holder>
    <shipping>
        <first-name>Paul</first-name>
        <last-name>Peterson</last-name>
        <address>
            <street1>978C Golf Street</street1>
            <city>Round Lake</city>
            <state>IL</state>
            <country>US</country>
            <postal-code>60073</postal-code>
        </address>
        <shipping-method>home_delivery</shipping-method>
    </shipping>
    <card>
        <expiration-month>1</expiration-month>
        <expiration-year>2023</expiration-year>
        <card-type>mastercard</card-type>
    </card>
    <card-token>
        <token-id>5489620654931192</token-id>
        <masked-account-number>541333******1192</masked-account-number>
    </card-token>
    <ip-address>127.0.0.1</ip-address>
    <order-detail>74b0d212-0ef8-6134-38ae-2e996a714972</order-detail>
    <descriptor>3DS-2 Use-Cases: Delayed Shipment - Expected Delay less than 90 days</descriptor>
    <notifications format="application/json">
        <notification url="http://www.webhook.site/68f1326e-311d-4ef8-a998-1c3c66cc497a"></notification>
    </notifications>
    <payment-methods>
        <payment-method name="creditcard"/>
    </payment-methods>
    <parent-transaction-amount currency="EUR">10.000000</parent-transaction-amount>
    <authorization-code>695054</authorization-code>
    <three-d>
        <cardholder-authentication-status>Y</cardholder-authentication-status>
    </three-d>
    <api-id>elastic-api</api-id>
    <provider-account-id>70003</provider-account-id>
</payment>